store/src/java/com/zimbra/cs/filter/jsieve/EditHeaderExtension.java

```diff
@@ -564,4 +566,28 @@ private boolean matchValue(String regex, String value) {
         Matcher matcher = pattern.matcher(value);
         return matcher.matches();
     }
+
+    static public boolean isTolerableFormedMessage(ZimbraMailAdapter zma, String actionName, MimeMessage mm) {
+        if (zma.getEditHeaderParseStatus() == PARSESTATUS.UNKNOWN) {
+            return saveChanges(zma, actionName, mm);
+        } else if (zma.getEditHeaderParseStatus() == PARSESTATUS.MIMEMALFORMED) {
+            ZimbraLog.filter.info(actionName + ": Triggering message is malformed MIME");
```

Comment:
should change this log level to debug level as for this mime, it has already been logged that it's malformed in saveChanges method. this would prevent multiple logging at info level for the same mime message.